pkg/cmd/generate-postgres-metadata-tables/main.go

```diff
@@ -100,3 +122,28 @@ func closeDB(conn *pgx.Conn) {
 		panic(err)
 	}
 }
+
+func getMappedType(dataType *string, dataTypeOid *uint32) error {
```

Comment:
i would prefer to change this so it doesn't modify the parameters. it can return the mappedTypeName and mappedTypeOID and an error, and change the parameter types so they are not pointers

maybe `dataType` should be renamed to `dataTypeName` here also

also, can you add a comment to the function?

Reply:
Done.